src/dlgPackageExporter.cpp

```diff
@@ -74,46 +70,64 @@ dlgPackageExporter::dlgPackageExporter(QWidget *parent, Host* host) :
     treeWidget->addTopLevelItem(mpKeys);
     treeWidget->addTopLevelItem(mpButtons);
 
-    closeButton = ui->buttonBox->addButton(QDialogButtonBox::Close);
-    exportButton = new QPushButton(tr("&Export"));
+    // This button has the RejectRole which causes the dialog to be rejected
+    // (and closed):
+    mCancelButton = ui->buttonBox->button(QDialogButtonBox::Cancel);
+
+    // This button has the ApplyRole which applies current changes but does NOT
+    // cause the dialog to close:
+    mExportButton = ui->buttonBox->button(QDialogButtonBox::Apply);
+    mExportButton->setText(tr("Export", "Text for button to perform the package export on the items the user has selected."));
 
-    ui->browseButton->hide();
     ui->filePath->hide();
-    ui->textLabel1->hide();
+    ui->label_exportFilePath->hide();
 
-    // reset zipFile and filePath from possible previous use
-    zipFile = filePath = QLatin1String("");
+    // reset mPackagePathFileName and mXmlPathFileName from possible previous use
+    mPackagePathFileName.clear();
+    mXmlPathFileName.clear();
+    QString profileName(mpHost->getName());
 
-    packageName = QInputDialog::getText(nullptr, tr("Package name"), tr("Package name:"));
-    if (packageName.isEmpty()) {
+    mPackageName = QInputDialog::getText(nullptr, tr("Package name"), tr("What do you wish to call the package:"));
+    if (mPackageName.isEmpty()) {
         return;
     }
-    QString packagePath = QFileDialog::getExistingDirectory(nullptr, tr("Where do you want to save the package?"), tr("Where do you want to save the package?"));
+
+    // The native windows dialog does not support displaying files - and as this
+    // code will clobber/overwrite an existing package with the same
```

Comment:
Could that be fixed? In a sepeate PR..

Reply:
Well it was highly desirable to fix another bug in the same line (where we did not set the starting location in the old code - "Where do you want to save the package?" is NOT a valid path!) and I really wanted to fix both issues at the same time...

Reply:
Not sure why you can't propose seperate PRs simultaneously. Agree on fixing both bugs.

My comment above refered to the clobber/overwrite of existing packages without notice.
Maybe I should heed my own advice and create a seperate issue for this bug though. :bulb: